modules/fb_instant_articles_api/fb_instant_articles_api.info.yml

```diff
@@ -1,6 +1,6 @@
 name: Facebook Instant Articles API
 type: module
-description: API module for Facebook Instant Articles.
+description: Implements entity hooks that use the Instant Articles API to delver articles to the Instant Article platform.
```

Comment:
Spellcheck nazi:  delver should be deliver (since this is going out to the Drupal world at large).